source/common/http/conn_manager_impl.h

```diff
@@ -189,6 +189,9 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
                         absl::string_view details) override {
       return filter_manager_.sendLocalReply(code, body, modify_headers, grpc_status, details);
     }
+
+    void sendGoAwayandClose() override { return connection_manager_.sendGoAwayandClose(); }
```

Comment:
What happens if a filter calls `sendGoAwayandClose()` but then returns Continue from a callback?

Reply:
When goAway is sent, no matter it is `Continue` or `Stopiteration`, we tends to abort the current stream, this lifetime management is suitable IMHO.